src/components/Match/renderMatch.js

```diff
@@ -92,66 +96,100 @@ function generateTeamfights(match) {
   return (match.teamfights || []).map(tf => computeTfData(tf));
 }
 
+// create a detailed history of each wards
+function generateWardLog(match) {
+  const computeWardData = (player, i) => {
+    const sameWard = _.curry((w1, w2) => w1.ehandle === w2.ehandle);
```

Comment:
if we're not using the benefit of this function (accepts multiple arguments or one at a time), why not just
```js
w1 => w2 => w1.ehandle === w2.ehandle;
```
I am not opposed to using this better implementation of currying but if it's never going to be used like that, why include it (unless we think it would be in the future)?